pymc/model.py

```diff
@@ -1084,9 +1085,12 @@ def add_coord(
         values : optional, array-like
             Coordinate values or ``None`` (for auto-numbering).
             If ``None`` is passed, a ``length`` must be specified.
+        fixed : bool
+            Whether dimensions created from coords should be fixed.
```

Comment:
FYI Fixed has could have a double meaning here, Fixed meaning immutable, or fixed meaning "fixed from broken. Suggest expanding the docstring. @michaelosthege let me know if this makes sense

```suggestion
            Whether dimensions created from coords should be fixed to an immutable value.
```